tiledb/sm/query/legacy/reader.cc

```diff
@@ -596,9 +595,9 @@ Status Reader::compute_range_result_coords(
               range_result_coords[r].end(),
               range_result_coords[r].size(),
               sort_layout));
-          this->throw_if_cancellation_requested();
+          this->throw_if_cancelled();
```

Comment:
No need for redundant `this` here and elsewhere.
```suggestion
          throw_if_cancelled();
```